cli/tests/integration_tests.rs

```diff
@@ -18,6 +18,20 @@ use tempfile::TempDir;
 use test_util as util;
 use tokio::task::LocalSet;
 
+#[test]
+fn dts_doc() {
+  let status = util::deno_cmd()
+    .arg("test")
+    .arg("--doc")
+    .arg("--unstable")
+    .arg(util::root_path().join("cli/dts"))
```

Comment:
I'd prefer to run explicitly only our own declaration files, otherwise we'd be running declarations from coming from TS. WDYT Casper?

Reply:
Yeah turns out they're not passing.
Moving this out to https://github.com/denoland/deno/pull/10707

Need a new test case for this issue.